src/Prima/Pyxis/Link.elm

```diff
@@ -71,12 +153,12 @@ isStandalone =
         Link.simple "Visit site" "https://www.prima.it"
 
 -}
-simple : String -> String -> Config
-simple label path =
-    Config (Configuration Simple label path Nothing)
+simple : String -> String -> Config msg
```

Comment:
Farei in modo che `href` non sia obbligatorio. A volte vogliamo creare dei componenti che sono dei Links visivamente ma poi reagiscono solo all'onClick. In questo modo si sarebbe obbligati a passare un `href = ""`.
Quindi aggiungere `withHref`

Reply:
Si, ha senso. Aggiungo l'attributo